### PR TITLE
Fix brackets in colored messages

### LINKED ISSUE #49: Badly formatted colored log messages
### Expected Behavior

All logs use the same format, regardless of color
### Actual Behavior

Colored (non-default) log messages do not have the brackets around the date, making the line badly formatted

Bug image
### Steps to Reproduce

Uh... I think it's pretty straight-forward.
### Other Information

OS: Windows 10
Git Commit: e014532

### PR DESCRIPTION
**Pull request by meyer9**
### Changes:
- Fixed #49
- Fixed transfer not using log

@OpenPoGo/maintainers

### CODE REVIEW

File: pokemongo_bot/logger.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def log(string, color='white'):
     if color not in colorHex:
         print('[' + time.strftime("%Y-%m-%d %H:%M:%S") + '] ' + string)
     else:
-        print(u'\033[' + colorHex[color] + time.strftime("%Y-%m-%d %H:%M:%S") + ' ' + string.decode('utf-8') + '\033[0m')
+        print(u'\033[' + colorHex[color] + "[" + time.strftime("%Y-%m-%d %H:%M:%S") + '] ' + string.decode('utf-8') + '\033[0m')
```
Comment (MaxLeiter) [inline pokemongo_bot/logger.py:20]: Why the inconsistent use of " and '?

Comment (meyer9) [inline pokemongo_bot/logger.py:20]: escape characters

**Comment by MaxLeiter** [comment]
Please rebase with master

**Comment by meyer9** [comment]
:+1: can squash when we merge.

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/52/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
:+1: if that's the case

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/52/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by jtdroste** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/52/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**